xml/System.Text/SpanLineEnumerator.xml

```diff
@@ -49,8 +49,7 @@ To get an instance of this type, use <xref:System.MemoryExtensions.EnumerateLine
       </ReturnValue>
       <Docs>
         <summary>Gets the line at the current position of the enumerator.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
```

Comment:
I don't think we're supposed to remove `<remarks>` items, even if they are left undocumented. The default string is `To be added.`.

Although, I did not see any CI failures. @gewarren should these two remark lines be brought back or can they stay removed?

Reply:
It will get re-added on the next CI job, so it's probably best not to delete it.